### PR TITLE
Add sqlite example

### PR DESCRIPTION
**Pull request by killercup**
This adds @Phrohdoh's sqlite example to the diesel core repo and its CI (Travis for now, AppVeyor currently doesn't test examples). This is only the 3rd of the three steps of the Getting Started guide. I'd leave splitting off the first two steps for another PR (good for newcomers).

**Before merging:** @Phrohdoh, you published this under the MIT license. Diesel itself is dual-licensed under MIT/Apache 2.0. Can you relicense/allow us to use your code under both licenses? I've copied your license file for now.

Fixes Phrohdoh/rs-diesel-sqlite#2
cc #376

### CODE REVIEW

**Comment by killercup** [comment]
Also TIL`git merge --allow-unrelated-histories` :)

**Comment by sgrif** [comment]
Should we add steps 1 and 2 as well? Looks fine to me otherwise modulo licensing issues.

**Comment by Phrohdoh** [comment]
@killercup Yes, I'll license this under MIT+Apache2!
Once this is merged I'll delete my repo entirely.

How is this only the third step?
That repo covers everything in the getting started guide.

**Comment by killercup** [comment]
Thanks!

It's "only" the third step as the third is the one that contains everything in the guide, the steps are cumulative. ;) Step one is "show posts binary", step two "add write posts binary", and step three "add publish and delete".

**Comment by killercup** [comment]
Changed the license, merging.

File: examples/sqlite/test_all [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -e
+
+export DATABASE_URL="DATABASE_URL=file:test.db"
```
Comment (sgrif) [inline examples/sqlite/test_all]: That's an odd database url. ;)

**Pull request merged**